modules/Bio/EnsEMBL/Compara/PipeConfig/DatachecksForRelease_conf.pm

```diff
@@ -0,0 +1,374 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::DatachecksForRelease_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DatachecksForRelease_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -division $COMPARA_DIV
+
+=head1 DESCRIPTION
+
+The pipeline configuration that runs compara datachecks for the current release database
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::DatachecksForRelease_conf;
+
+
+use strict;
+use warnings;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf; # for WHEN and INPUT_PLUS
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::ComparaGeneric_conf');
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options},
+
+        division      => undef,
+        pipeline_name => $self->o('division').'_datachecks_'.$self->o('rel_with_suffix'),
+        registry_file => $self->o('reg_conf'),
+        db_type       => 'compara',
+        compara_db    => 'compara_curr',
+
+        history_file       => '/nfs/panda/ensembl/production/datachecks/history/compara.json',
+        output_dir_path    => $self->o('pipeline_dir').'/tapfile',
+        datacheck_groups   => ['compara'],
+        datacheck_types    => ['critical'],
+        datacheck_names    => [],
+        datacheck_patterns => [],
+        species            => [],
```

Comment:
This parameter is not used anywhere (not even declared as pipeline-wide parameter).